### DISCUSSION TITLE
tasmota32-bluetooth.bin & tasmota32-display.bin together

### DISCUSSION THREAD

**Discussion by zackfuchtel** [arendst/Tasmota#16730] [Q&A] [answered]
Hello,

is there a version available with both features?

tasmota32-bluetooth.bin = The Bluetooth version adds BLE support for 4M+ flash.
tasmota32-display.bin = The Display version without Energy Monitoring but adds display support for 4M+

Thanks.

**Comment by octavvian** [comment, marked as answer]
Hi, I think you can try to compile it you're self.
https://github.com/benzino77/tasmocompiler
search ”Super easy way (Thanks to @meingraham and @Jason2866)”... that's how I'm doing.
Just a suggestion...

**Reply by zackfuchtel** [reply]
Thank you, working with a ttGO-esp32 with display. Keep healthy.

**Reply by whateverthatis** [reply]
Hi, could you please show me how did you make the display work with the bluetooth? I compiled with TasmoCompiler with the selected features Bluetooth and Display for my Sonoff TH Elite, but no luck in seeing anything on the screen. Separately it does work, so either only tasmota32-bluetooth.bin or only tasmota32-display.bin but not the two together. Thank you.
